src/Data/Vec/Base.agda

```diff
@@ -263,16 +263,13 @@ splitAt : ∀ m {n} (xs : Vec A (m + n)) →
           ∃₂ λ (ys : Vec A m) (zs : Vec A n) → xs ≡ ys ++ zs
 splitAt zero    xs                = ([] , xs , refl)
 splitAt (suc m) (x ∷ xs)          with splitAt m xs
-splitAt (suc m) (x ∷ .(ys ++ zs)) | (ys , zs , refl) =
-  ((x ∷ ys) , zs , refl)
+splitAt (suc m) (x ∷ xs) | (ys , zs , p) = ((x ∷ ys) , zs , cong (x ∷_) p)
```

Comment:
Don't need the outermost brackets on the LHS?

Reply:
This is coming from #2056. I will rebase once that is merged. (Some of the proofs in this PR depend on #2056.)

Reply:
Nor on the RHS... ;-)